docs/typescript/generating-types.mdx

```diff
@@ -18,6 +18,32 @@ payload generate:types
 
 You can run this command whenever you need to regenerate your types, and then you can use these types in your Payload code directly.
 
+### Disable declare statement
+
+By default, `generate:types` will add a `declare` statement to your types file in order to better assist with type inference.
+
+If you would like to disable this, you can add the `declare` property as false under the `typescript` settings.
+
+```ts
+// payload.config.ts
+{
+  // ...
+  typescript: {
+    declare: false, // defaults to true if not set
+  },
+}
+```
+
+You can then manually add the `declare` statement to your types file if you wish in the following format:
```

Comment:
If you do disable the `declare` pattern, you'll need to manually add a `declare` statement to your code in order for Payload types to be recognized. Here's an example showing how to declare your types in your `payload.config.ts` file: